plugins/woocommerce/src/Internal/CLI/Migrator/Commands/ResetCommand.php

```
@@ -0,0 +1,38 @@
+	 *
+	 * [--platform=<platform>]
+	 * : The platform to reset credentials for. Defaults to 'shopify'.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     wp wc migrate reset
+	 *
+	 * @param array $args       Positional arguments.
+	 * @param array $assoc_args Associative arguments.
+	 */
+	public function __invoke( array $args, array $assoc_args ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (Unused Code Rules)

(UnusedFormalParameter)